raiden-dapp/src/components/TokenOverlay.vue

```diff
@@ -124,6 +120,15 @@ export default class TokenOverlay extends Mixins(
   allTokens!: Token[];
   tokens!: TokenModel[];
 
+  handleTokenClick(tokenAddress: string) {
+    const { token } = this.$route.params;
+    if (token === tokenAddress) {
+      this.cancel();
```

Comment:
@nephix I am curious about this change, could you describe to be what it does and what problem it solves, please?

Reply:
Does it close the overlay if you try to navigate to the token you already are?

Reply:
It was a bug that I found when trying to click/copy the token address in the TokenOverlay component. The `to=...` declaration [here](https://github.com/raiden-network/light-client/pull/911/files#diff-e66fa633bbb3802562485bba974d68eaL53) did not prevent the event bubbling when trying to copy the address so I had to remove it. 

And yes, the 2nd issue was that it redirected even if you have clicked on the token you are currently on (instead of just closing the overlay)

Reply:
Good finds, thank you :+1: